accounts.go

```diff
@@ -0,0 +1,109 @@
+package recurly
+
+import (
+	"encoding/xml"
+	"time"
+
+	"github.com/blacklightcms/go-recurly/types"
+)
+
+type (
```

Comment:
Fixed in [b83c87](https://github.com/blacklightcms/go-recurly/pull/10/commits/e775de18bcd2e4a91905e2431ef00d2d33b83c87)